Add CF Turnstile captcha

boba1007 opened the pull request.

boba1007 committed:
Add CF Turnstile captcha

ajohn25 commented:
@boba1007 sorry for long delay on reviewing this 😭 is this still relevant to review now that we've switched over to the mailer form?

boba1007 commented:
> @boba1007 sorry for long delay on reviewing this 😭 is this still relevant to review now that we've switched over to the mailer form?

Yes, we updated quick signup to use Mailer forms, but this still applies to the other lead forms to reduce spams as well

ajohn25 commented on src/pages/api/send-email.ts:
```diff
@@ -67,7 +96,7 @@ export const POST = async ({
 					subdomain,
 					billingAddress,
 					needs,
-					timeline: timelineLabel, // using the label here
```
lol

ajohn25 reviewed (approved):
thx for confirming and sorry again about delay!
